bedrock/careers/templates/careers/includes/testimonials.html

```diff
@@ -14,7 +14,7 @@
           Read full quote
           </a>
         </li>
-        <li class="c-careers-testimonial-image"></li>
+        <li class="c-careers-testimonial-image perla"></li>
```

Comment:
**Suggestion:** Since the images + quotes are going to be rotating from time to time, let's not use classnames that reflect the people in the images, but rather generic classes that we don't have to change whenever we choose to update the images.
i.e. (`.c-testimonial-image-one`)